prime-router/src/main/kotlin/cli/TestReportStream.kt

```diff
@@ -734,56 +757,107 @@ class Waters : CoolTest() {
     override val description = "Submit data in waters schema, send to BLOBSTORE only"
     override val status = TestStatus.GOODSTUFF
 
-    override fun run(environment: ReportStreamEnv, items: Int, submits: Int, key: String?, dir: String): Boolean {
-        ugly("Starting Waters Test: sending Waters data to the ${blobstoreReceiver.name} receiver only.")
+    override fun run(environment: ReportStreamEnv, options: CoolTestOptions): Boolean {
+        ugly("Starting Waters: sending ${options.items} Waters items to ${blobstoreReceiver.name} receiver")
         val file = FileUtilities.createFakeFile(
             metadata,
             watersSender,
-            items,
+            options.items,
             receivingStates,
             blobstoreReceiver.name,
-            dir,
+            options.dir,
         )
         echo("Created datafile $file")
-        val (responseCode, json) = HttpUtilities.postReportFile(environment, file, org.name, watersSender.name, key)
+        val (responseCode, json) =
+            HttpUtilities.postReportFile(environment, file, org.name, watersSender.name, options.key)
         echo("Response to POST: $responseCode")
-        echo(json)
+        if (!options.muted) echo(json)
         if (responseCode != HttpURLConnection.HTTP_CREATED) {
             return bad("***Waters Test FAILED***:  response code $responseCode")
         }
         val tree = jacksonObjectMapper().readTree(json)
         val reportId = ReportId.fromString(tree["id"].textValue())
         echo("Id of submitted report: $reportId")
-        waitABit(25, environment)
-        return examineLineageResults(reportId, listOf(blobstoreReceiver), items)
+        waitABit(25, environment, options.muted)
+        if (file.exists()) file.delete()  // because of RepeatWaters
+        return examineLineageResults(reportId, listOf(blobstoreReceiver), options.items)
     }
 }
 
+class RepeatWaters : CoolTest() {
```

Comment:
All the other changes in this file are to support this change:   Adding this new `repeatwaters` test, which repeatedly sends submits to the waters schema.  

As set at the moment, it sends a new Report every 360ms, with a little randomization thrown in.   So if you put 10 Items in each report, that's a pace of 100,000 Items per hour.     This appears to be a sustainable pace on localhost - I ran this with no problems:
` ./prime test --run repeatwaters --submits 2500 --items 10`

This test differs from hammertime in that hammer time submits all its reports all at once, whereas this is a more realistic scenario with reports coming in at random times over an extended time period.